pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css

```diff
@@ -0,0 +1,16 @@
+.sponsor-logo-with-text {
+  display: flex;
+  justify-content: center;
+  font-size: 24px;
+  margin-top: 30px;
+  clear: both;
+  padding-top: 30px;
+}
+
+.sponsor-subtext-description {
+  font-size: 14px;
+  text-align: center;
+  padding-top: 20px;
+  margin-left: 5%;
```

Comment:
The % here might get wonky when you check responsiveness on mobile; you can leave for now and adjust later.

Reply:
Good point!  I am curious as to why px work better when checking for responsiveness on mobile?

Reply:
Let's say the full width is 1000px, and margin-left of 5% means you'd have 50px on the left of the element.  As your screen size shrinks, 5% of a smaller screen size is much smaller than the margin you had before.  This effect may or may not be what you want to have happen, so it's good to check various screen sizes.